felix/bpf/conntrack/scanner.go

```diff
@@ -156,14 +167,29 @@ func (s *Scanner) Scan() {
 				"entry": ctVal,
 			}).Debug("Examining conntrack entry")
 		}
+		val = cleanupv1.Value{}
+		if s.ipVersion == 6 {
+			val = cleanupv1.ValueV6{}
+		}
 
 		for _, scanner := range s.scanners {
-			if verdict := scanner.Check(ctKey, ctVal, s.get); verdict == ScanVerdictDelete {
+			if verdict, ts := scanner.Check(ctKey, ctVal, s.get); verdict == ScanVerdictDelete {
 				if debug {
 					log.Debug("Deleting conntrack entry.")
 				}
-				cleaned++
-				return maps.IterDelete
```

Comment:
Have we lost the option to fall back to user-space only?  ISTR that BPF cleaner needs a new-ish kernel; we should make sure we know what kernel is needed and get sign off if we're effectively ending support for older versions.

Then we should release note it.

If user-space is no longer supported, we should deprecate the config.

Reply:
```BPFConntrackCleanupMode``` is already deprecated and defaulted to userspace.